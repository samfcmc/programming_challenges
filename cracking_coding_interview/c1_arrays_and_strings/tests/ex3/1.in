aaaaaaaaaaaaaa
baaaaaaaaaaaab
bcdefg
hello
lalala
aabbcc
abbc
