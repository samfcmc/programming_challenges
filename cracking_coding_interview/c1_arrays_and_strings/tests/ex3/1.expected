a
ba
bcdefg
helo
la
abc
abc
